### PR TITLE
[voice] Remove add-ons porcupineks and actiontemplatehli.

### PR DESCRIPTION
**Pull request by GiviMAD**
Remove porcupineks and actiontemplatehli from add-on config.

### CODE REVIEW

**Comment by GiviMAD** [comment]
@lolodomo, I'm sorry, I didn't understand that I should do this.

Do you think I need to add any alert messages?

**Comment by lolodomo** [comment]
> Do you think I need to add any alert messages?

Yes, it would be probably better to clearly announce it.

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -247,3 +249,11 @@ REPLACE;imperihome;;$OPENHAB_USERDATA/config/org/openhab/addons.config
...
 REPLACE;darksky,;;$OPENHAB_USERDATA/config/org/openhab/addons.config
 REPLACE;darksky;;$OPENHAB_USERDATA/config/org/openhab/addons.config
+
+[4.1.0]
+REPLACE;actiontemplatehli,;;$OPENHAB_USERDATA/config/org/openhab/addons.config
+REPLACE;,actiontemplatehli;;$OPENHAB_USERDATA/config/org/openhab/addons.config
+REPLACE;actiontemplatehli;;$OPENHAB_USERDATA/config/org/openhab/addons.config
+REPLACE;,porcupineks;;$OPENHAB_USERDATA/config/org/openhab/addons.config
+REPLACE;porcupineks,;;$OPENHAB_USERDATA/config/org/openhab/addons.config
+REPLACE;porcupineks;;$OPENHAB_USERDATA/config/org/openhab/addons.config
```
Comment (J-N-K) [inline distributions/openhab/src/main/resources/bin/update.lst:259]:
```suggestion
REPLACE;porcupineks;;$OPENHAB_USERDATA/config/org/openhab/addons.config

```

**Review by J-N-K — APPROVED** [review]
Thanks!

**Comment by GiviMAD** [comment]
Thank you!

**Pull request merged**